cypress/e2e/yves/claim/claim.cy.ts

```diff
@@ -0,0 +1,313 @@
+        withoutSession: true,
+      });
+
+      orderDetailPage.visit({
+        qs: {
+          id: dynamicFixtures.order.id_sales_order,
+        },
+      });
+      claimOrderPage.clickCreateClaimButton();
+
+      claimCreatePage.assertPageLocation();
+      staticFixtures.orderClaim.availableTypes = staticFixtures.claimTypes.order;
+      claimCreatePage.createOrderClaim({
+        ...staticFixtures.orderClaim,
+        orderReference: dynamicFixtures.order.order_reference,
+      });
+
+      claimDetailPage.assertPageLocation();
+      cy.contains(claimCreatePage.getClaimCreatedMessage()).should('exist');
+
+      var reference: string;
+      cy.url()
+        .then((url) => {
+          const urlParams = new URLSearchParams(url.split('?')[1]);
+          reference = urlParams.get('reference');
+        })
+        .then(() => {
+          claimDetailPage.assertOrderClaimDetails({
+            reference: reference,
+            type: staticFixtures.orderClaim.type,
+            subject: staticFixtures.orderClaim.subject,
+            description: staticFixtures.orderClaim.description,
+            status: staticFixtures.orderClaim.status,
+            files: staticFixtures.orderClaim.files,
+            date: new Date()
+              .toLocaleDateString('en-US', {
+                year: 'numeric',
+                month: 'short',
+                day: '2-digit',
+              })
+              .replace(/([a-zA-Z]+)\s/, '$1. '),
+            customer: {
+              firstName: dynamicFixtures.customer.first_name,
+              lastName: dynamicFixtures.customer.last_name,
+              email: dynamicFixtures.customer.email,
+              companyName: dynamicFixtures.company.name,
+              businessUnitName: dynamicFixtures.businessUnitFromCompany.name,
+            },
+            orderReference: dynamicFixtures.order.order_reference,
+          });
+        });
+    });
+
+    it('customer should be able to cancel a general claim', (): void => {
+      customerLoginScenario.execute({
+        email: dynamicFixtures.customer.email,
+        password: staticFixtures.defaultPassword,
+        withoutSession: true,
+      });
+
+      claimListPage.visit();
+
+      claimListPage.clickCreateClaimButton();
+
+      claimCreatePage.createClaim(staticFixtures.generalClaim);
+
+      claimDetailPage.assertPageLocation();
+      claimDetailPage.clickCancelClaimButton();
+
+      cy.get(claimDetailPage.getPendingClaimStatusSelector()).should('exist');
+    });
+
+    it('customer should not be able to cancel a claim if he is now owner', (): void => {
+      customerLoginScenario.execute({
+        email: dynamicFixtures.customer.email,
+        password: staticFixtures.defaultPassword,
+        withoutSession: true,
+      });
+
+      claimListPage.visit();
+
+      claimListPage.clickCreateClaimButton();
+
+      claimCreatePage.createClaim(staticFixtures.generalClaim);
+
+      claimDetailPage.assertPageLocation();
```

Comment:
no need to assert page location as we have covered it in another test. this one is about permissions